pkg/util/metrics/problem_counter.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package metrics
+
+import (
+	"context"
+	"fmt"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+type ProblemCounter struct {
+	metricName string
+	keyReason  tag.Key
+	counter    *stats.Int64Measure
+}
+
+func NewProblemCounter(metricName string, description string) (ProblemCounter, error) {
+	pc := ProblemCounter{metricName: metricName}
+
+	var err error
+	pc.keyReason, err = tag.NewKey("reason")
+	if err != nil {
+		return pc, err
+	}
+	pc.counter = NewInt64Metric(
+		pc.metricName,
+		description,
+		"",
```

Comment:
Oh you are right, it should be "1".


> How about defining a common type as a const in metrics package and invoke here?

Do you mean something like below?
```
const unitCount = "1"
const unitSecond = "second"
```

I feel it's a little redundant. Today, we have `"1"` and [`"second"`](https://github.com/kubernetes/node-problem-detector/blob/146dfd70b25090203162b1e9eee305e76b688dff/pkg/systemstatsmonitor/disk_collector.go#L52). Soon, we will have "kB", "kB/s" and others.

I slightly prefer to just keep them defined at where the metrics are defined, rather than in a central `metrics` package. I don't have a strong preference though. WDYT?

Reply:
Sounds good.